lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,58 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
+    collection.map{|elem| elem.is_a?(Numeric)}.count(true)
   end
 
   # Refer to: all?
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-all-3F
   def all_words_longer_than_length?(min_length)
+    collection.all? { |word| word.length > min_length}
   end
 
 
   # For the next 3 methods refer to: collect or map
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-any-3F
   def contains_a_word_longer_than?(min_length)
+    collection.map { |word| word.length > min_length }[-1]
```

Comment:
Seems like the commented instructions were a bit confusing here. If you use any? instead of map, you don't have to have the [-1] on the end.

`collection.any? { |word| word.length > min_length }`